### DISCUSSION TITLE
Ability to send message in a thread as well as channel

### DISCUSSION THREAD

**Discussion by darkLord19** [mattermost/community#105] [Practical Ideas]
With CRT being in beta currently and in stable in near future, user should have ability to reply in a thread and if required send to channel. There are many uses when one would want to bring attention to particular message of thread.

**Comment by chetanyakan** [comment]
These messages posted to the channel should also have the ability to start threads of their own.

**Reply by darkLord19** [reply]
That'd be tricky though as it would mostly involve nested threading because the post inside thread and in channel will be same but it will be displayed in two places

**Comment by esethna** [comment]
Ya I think there's two ideas here:

1. Popping out a thread to start a new thread based on that message
2. Getting the attention of the channel for specific messages in a thread

Separate ideas, but both certainly valid!
